gsa/src/web/entity/useExportEntity.js

```diff
@@ -0,0 +1,57 @@
+/* Copyright (C) 2020 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+import {useCallback} from 'react';
+
+import {useSelector} from 'react-redux';
+
+import {capitalizeFirstLetter} from 'gmp/utils/string';
+
+import {getUserSettingsDefaults} from 'web/store/usersettings/defaults/selectors';
+
+import {generateFilename} from 'web/utils/render';
+import useUserName from 'web/utils/useUserName';
+
+export const useExportEntity = () => {
```

Comment:
I wonder if we should also use the default export here same as for usePagination?